README.rst

```diff
@@ -12,6 +12,15 @@ Driver Mantras
 When developing specifications -- and the drivers themselves -- we follow the
 following principles:
 
+Strive to be idiomatic, but favor consistency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Drivers attempt to provide the easiest way to work with MongoDB in a given 
+language ecosystem while specifications attempt to provide a consistent 
```

Comment:
"language ecosystem, while specifications"

Reply:
Thanks, Matt! Added the comma for clarity/correctness.